CollAction/Controllers/ProjectsController.cs

```diff
@@ -99,7 +99,8 @@ public ViewResult StartInfo()
             return View(new CreateProjectViewModel
             {
                 Categories = new SelectList(await _context.Categories.ToListAsync(), "Id", "Description"),
-                Locations = new SelectList(await _context.Locations.ToListAsync(), "Id", "Name", null)
+                // TODO: Add this back in when we've come up with a solution that doesn't involve returning the entire locations table!!!
```

Comment:
You can probably remove this entirely. The final solution will probably involve an API endpoint. We'll likely forget removing stuff like this when we're adding an API endpoint somewhere else.